src/libs/ReportUtils.js

```diff
@@ -1255,7 +1255,8 @@ function getMoneyRequestReportName(report, policy = undefined) {
  * @returns {Object}
  */
 function getReport(reportID) {
-    return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {});
+    // Deleted reports are set to null and lodashGet will still return null in that case, so we need to add an extra check
+    return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {}) || {};
```

Comment:
Why are we returning `{}` for missing reports? 😬 wouldn't `null` or `undefined` be better in these cases?

Reply:
Taking my comment to the original PR that added this function (ignore this here)

https://github.com/Expensify/App/pull/24345/files#r1335355086